Update network request guide to include info about disabling logging (#5124)
cypress-io/cypress-documentation: docs/guides/guides/network-requests.mdx

## Command Log

> By default Cypress logs all `XMLHttpRequest`s and `fetch`es made by the

Reviewer: ```suggestion
By default, Cypress logs all `XMLHttpRequest`s and `fetch`es made by the
```

  alt="Screenshot of fetch and XHR requests"
/>

:::info

> **Note:** Logging can be disabled off by passing `{ log: false }` with your

Reviewer 2: ```suggestion
**Note:** Logging can be disabled by passing `{ log: false }` with the
```

static response.

```js

> cy.intercept('/users*', { body: ['user1', 'user2'], log: false }).as('getUsers')

Reviewer 2: Should we use the example that disables all xhr/fetch logs?

https://docs.cypress.io/api/commands/intercept#Disabling-logs-for-a-request

Author: I specifically used the example from farther down in the code to match: https://github.com/cypress-io/cypress-documentation/pull/5124/files#diff-bb8ce895b7660f9f324902a3e98c85d05f942fdd16997394dd5fc0dc46abb095L554

but could change to disable all if people want.

Reviewer 2: Could also link out to the example that shows how to disable all if we didn't want to update here.

Author: updated in c020484
